### PR TITLE
Add a draft CoC (Contributor Covenant v1.4.1)

### PR DESCRIPTION
**Pull request by brianwarner**
Adding a draft Code of Conduct for review and approval by the project.  This
commit contains the text of the Contributor Covenant v1.4.1.

This will be reviewed in the first TSC meeting.

Signed-off-by: Brian Warner <brian@bdwarner.com>

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (caniszczyk) [inline CODE_OF_CONDUCT.md]: @brianwarner can you create a codeofconduct@prestodb.io

**Comment by caniszczyk** [comment]
@brianwarner can you create a codeofconduct@prestodb.io email list?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (brianwarner) [reply CODE_OF_CONDUCT.md]: Sure, I've created this list: codeofconduct@lists.prestodb.io

This is a list with private archives, and the only members will be the moderation team responsible for reviewing code of conduct reports.  In order to mail this list with a report directly, a submitter must already be a member of at least one PrestoDB mailing list.  Any mails sent to the list by non-members will be held until a moderator can confirm they aren't spam, and forward them to the team.  This is all described on the list's main page: https://lists.prestodb.io/g/codeofconduct

```suggestion
reported by contacting the project team at [codeofconduct@lists.prestodb.io](codeofconduct@lists.prestodb.io). All
```

**Comment by brianwarner** [comment]
@caniszczyk how would you feel about this file moving over the the (newly created) .github repo, so that it replicates across the organization when it lands?

**Comment by caniszczyk** [comment]
@brianwarner that's fine, let's just get it done and approved :)

**Pull request merged**